src/headers.js

```diff
@@ -3,19 +3,19 @@ export class Headers {
     this.headers = headers;
   }
 
-  add(key, value){
```

Comment:
It'd be great if type annotations were consistently filled in on all methods. Eg, this one is missing a type on `value`, and the return type.